plugins/source/datadog/codegen/recipies/base.go

```diff
@@ -0,0 +1,112 @@
+package recipies
```

Comment:
```suggestion
package recipes
```

This package should be renamed to `recipes` to fix the typo